docs/source/modules/damping_driver.rst

```diff
@@ -0,0 +1,139 @@
+| ``sgsmtn``            | sub-grid scale topography variance |:math:`m`               |
++-----------------------+------------------------------------+------------------------+
+
+**Convective GWD:**
+
++-----------------------+------------------------------------+------------------------+
+| Name                  | Description                        | Units                  |
+|                       |                                    |                        |
++=======================+====================================+========================+
+| ``udt_cgwd``          | u wind tendency                    |:math:`m s^{-2}`        |
++-----------------------+------------------------------------+------------------------+
+
+**Constant Drag:**
+
++-----------------------+------------------------------------+------------------------+
+| Name                  | Description                        | Units                  |
+|                       |                                    |                        |
++=======================+====================================+========================+
+| ``udt_cnstd``         | u wind tendency                    |:math:`m s^{-2}`        |
++-----------------------+------------------------------------+------------------------+
+
+**topo_drag:**
+
+Note: These are not currently available
+
++-----------------------+------------------------------------+------------------------+
+| Name                  | Description                        | Units                  |
+|                       |                                    |                        |
++=======================+====================================+========================+
+| ``udt_topo``          | u wind tendency                    |:math:`m s^{-2}`        |
++-----------------------+------------------------------------+------------------------+
+| ``vdt_topo``          | v wind tendency                    |:math:`m s^{-2}`        |
++-----------------------+------------------------------------+------------------------+
+
+Relevant modules and subroutines
+--------------------------------
+The code is split into 4 subroutines; ``damping_driver``, ``damping_driver_init``, ``damping_driver_end`` and ``rayleigh``. The ``_init`` and ``_end`` subroutines are for initializing and closing the module. The majority of the ``damping driver`` code is just a switchboard with the exeption of the Rayleigh and constant drag calculations. The calculations for the other drag schemes are given in their own documentation.
+
+**Rayleigh Drag**
+
+Located in the ``rayleigh`` subroutine. This code damps the momentum tomward zero in the upper model levels specified. The zonal/meridional tendency for each grid is calculated my multiplying the zonal/meridional velocity by a factor determined by the pressure and rayleigh parameters. The higher the wind velocity, the more the damping.
```

Comment:
typo: towards
I'd write: specified upper model levels.
typo: by
Edit last sentence: The damping is proportional to the wind velocity.

Reply:
Much better thank you! I also added "cell"